source/include/azure_iot_version.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot_version.h
+ *
+ * @brief Provides version information.
```

Comment:
Nit - "version information for Azure IoT FreeRTOS middleware" or whatever the right terminology we need to use here.